### ISSUE TITLE
NODE USER FEEDBACK SESSION- 2019-04-25 - Virtual - Node Enterprise Focus Group

### ISSUE THREAD

**Issue by ahmadnassri** [nodejs/user-feedback#144]
## Date & Time

**UTC Thursday 25-April-2019 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 25-Apr-2019 09:00 (09:00 AM) |
| US / Mountain | Thu 25-Apr-2019 10:00 (10:00 AM) |
| US / Central  | Thu 25-Apr-2019 11:00 (11:00 AM) |
| US / Eastern  | Thu 25-Apr-2019 12:00 (12:00 PM) |
| London        | Thu 25-Apr-2019 17:00 (05:00 PM) |
| Amsterdam     | Thu 25-Apr-2019 18:00 (06:00 PM) |
| Moscow        | Thu 25-Apr-2019 20:00 (08:00 PM) |
| Chennai       | Thu 25-Apr-2019 22:30 (10:30 PM) |
| Hangzhou      | Fri 26-Apr-2019 01:00 (01:00 AM) |
| Tokyo         | Fri 26-Apr-2019 02:00 (02:00 AM) |
| Sydney        | Fri 26-Apr-2019 04:00 (04:00 AM) |

Or in your local time:

* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node%20Enterprise%20Focus%20Group+2019-04-25&iso=20190425T17

## Links

* Minutes Google Doc: <tbd>

## Agenda

Gather feedback and insights on challenges faced by closed-source projects that use Node.js within the context of large businesses. 
These challenges might not be immediately visible to the open source community, but they have a domino effect on all users. 

* The effects of change in the OSS domain to business operations
* The cost of adoption vs. the total cost of ownership for Node.js application development
* The capacity for Node.js to drive business revenue and curb cost burdens

## Invited

* User Feedback Members: @nodejs/user-feedback

## Joining the meeting

* link for participants: https://zoom.us/j/353642832
* We stream our conference call straight to YouTube so anyone can listen to it live.
  It should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on.
  There's usually a short cat-herding time at the start of the meeting and then occasionally we
  have some quick private business to attend to before we can start recording & streaming.
  So be patient and it should show up.

**Comment by boneskull** [comment]
Do we have participants lined up for this?

**Comment by ahmadnassri** [comment]
@boneskull yes, I'll be updating the ticket with their names and making a PR to the repo as well, once I get confirmation from them.

**Comment by ShrawanLakhe** [comment]
how can one participate in these sessions?

**Comment by ahmadnassri** [comment]
@ShrawanLakhe you can join the meeting as a participant on Zoom: https://zoom.us/j/353642832, or watch live stream on YouTube: https://www.youtube.com/c/nodejs+foundation/live

if you wish to become a member / active participant, then that is a good question, and I do not have a good answer to how-to yet, perhaps that would be a topic of discussion in the meeting itself. _(i.e. how to systematically invite / include representation from more enterprise companies)_

thoughts and feedback more than welcome as we build out this focus group!

**Comment by ShrawanLakhe** [comment]
> @ShrawanLakhe you can join the meeting as a participant on Zoom: https://zoom.us/j/353642832, or watch live stream on YouTube: https://www.youtube.com/c/nodejs+foundation/live
> 
> if you wish to become a member / active participant, then that is a good question, and I do not have a good answer to how-to yet, perhaps that would be a topic of discussion in the meeting itself. _(i.e. how to systematically invite / include representation from more enterprise companies)_
> 
> thoughts and feedback more than welcome as we build out this focus group!

@ahmadnassri Thank you for the info

**Comment by mihaiep** [comment]
Thank you, Ahmad! blog post is live: https://medium.com/@nodejs/introducing-the-node-js-enterprise-focus-group-312ff9f49aaa

**Comment by dominykas** [comment]
The headline here says the meeting time is:

> UTC Thursday 25-April-2019 17:00 (05:00 PM):

But the table says:

> London | Thu 25-Apr-2019 17:00 (05:00 PM)

I assume the UTC one is correct (i.e. it's 6pm in London, due to summer time), so the table should be updated?

Also is it intended to overlap with the meeting in #149?

**Comment by mhdawson** [comment]
@ahmadnassri can you clear up the confusion over the time?  We either need to change this issue or the calendar entry.

**Comment by ahmadnassri** [comment]
updated the time table, thank you @dominykas for catching that mistake, a couple of other cities were also fixed!

also, to note this meeting replaces #149

**Comment by mhdawson** [comment]
@ahmadnassri I'm still confused the Calendar say 1 PM, while the entries in the first post says 12 EST in the list of times.  I'm assuming its 1EST since that is what is in the calendar and matches 5 PM UTC but the table of times in the first post seems to be out of date.

**Comment by ahmadnassri** [comment]
@mhdawson we should (if this doesn't exist already) talk about automating the issue creation with some cli tooling to solve these manual steps ;)

hopefully fixed the time table for the last time, thank you all!

**Comment by mhdawson** [comment]
@ahmadnassri there is already automated generation.  The other issue that we closed was the automatically generated issue.  It works best for recurring meetings and automatically generates the issue from the node.js/calendar.  Once there is a cadence for the enterprise sub-group you can PR templates into: https://github.com/nodejs/create-node-meeting-artifacts/tree/master/templates as per https://github.com/nodejs/create-node-meeting-artifacts/blob/master/TEMPLATES_DOCUMENTATION.md

**Comment by ahmadnassri** [comment]
this is good to know! thank you @mhdawson

**Comment by mihaiep** [comment]
live now: https://www.youtube.com/watch?v=PhyHkdoQDac

**Comment by dshaw** [comment]
Node.js release schedule JSON https://raw.githubusercontent.com/nodejs/Release/master/schedule.json

**Comment by mihaiep** [comment]
Thank you for participating in this meeting. Meeting notes in a couple of hours will be available here:
https://docs.google.com/document/d/15RDN7deM6wf2X271zpayrRSSnqdGO_1S0kTvfx5Agwo
